app/components/chat-message.js

```diff
@@ -3,7 +3,7 @@ import { computed } from '@ember/object';
 
 export default Component.extend({
   gifsEnabled: true,
-  imgRegex: /https?:\/\/(?:[a-z0-9\-]+\.)+[a-z]{2,6}(?:\/[^\/#?]+)+\.(?:jpg|gif|png|webp)(\?.*$)*/,
+  imgRegex: /https?:\/\/(?:[a-z0-9-]+\.)+[a-z]{2,6}(?:\/[^/#?]+)+\.(?:jpg|gif|png|webp)(\?.*$)*/,
```

Comment:
Going to double check this works....

Reply:
seems good
![img_regex_test](https://user-images.githubusercontent.com/66243/68262646-3cde2d80-0087-11ea-828b-e6631cd3a42d.JPG)